roles/transformers/templates/alfresco-tengine-aio.service.j2

```diff
@@ -0,0 +1,65 @@
+{%- macro ats_proto() -%}
+http
+{%- if ats_keystore.cert_containers | default([]) | length > 0 -%}
+s
+{%- endif -%}
+{%- endmacro %}
+
+{%- macro im_lib() -%}
+{%- if pkg_format == 'deb' -%}
+{{ imagemagick_home }}/lib/x86_64-linux-gnu/
+{%- else -%}
+/usr/lib64/
+{%- endif -%}
+{%- endmacro %}
+
+{%- macro im_cfg() -%}
+{%- if pkg_format == 'deb' -%}
+{{ imagemagick_home }}/lib/x86_64-linux-gnu/ImageMagick-{{ imagemagick_version.split('-')[0] -}}
+{%- else -%}
+{{ imagemagick_home }}
+{%- endif -%}
+{%- endmacro %}
+
+{%- macro standrad_output_macro() -%}
+{%- if pkg_format == 'deb' -%}
```

Comment:
probably better to use `ansible_os_family` or `ansible_distribution` if needs to be more specific